catalog/plugin/catalog-plugin-clavin/src/test/java/org/codice/ddf/catalog/content/plugin/clavin/TestClavinWrapper.java

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.content.plugin.clavin;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.core.Is.is;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import com.bericotech.clavin.ClavinException;
+import com.bericotech.clavin.index.IndexDirectoryBuilder;
+import com.bericotech.clavin.resolver.ResolvedLocation;
+
+public class TestClavinWrapper {
+
+    @Rule
+    public ExpectedException expectedEx = ExpectedException.none();
+
+    private static final String RESOURCE_PATH =
```

Comment:
You don't need to do this. Resources are loaded at "/" inside the test jar. You should be using a temporary directory rule in Junit and copying any files out of the resources that you want to mess with to that temp area and then working with them there.